{
	"about": "https://jvndb.jvn.jp/ja/contents/2020/JVNDB-2020-018046.html",
	"title": "easy-parse project の Python 用 easy-parse における XML 外部エンティティの脆弱性",
	"link": "https://jvndb.jvn.jp/ja/contents/2020/JVNDB-2020-018046.html",
	"description": "easy-parse project の Python 用 easy-parse には、XML 外部エンティティの脆弱性が存在します。",
	"identifier": "JVNDB-2020-018046",
	"references": [
		{
			"text": "https://www.cve.org/CVERecord?id=CVE-2020-26710",
			"source": "CVE",
			"id": "CVE-2020-26710"
		},
		{
			"text": "https://nvd.nist.gov/vuln/detail/CVE-2020-26710",
			"source": "NVD",
			"id": "CVE-2020-26710"
		},
		{
			"text": "https://github.com/uncmath25/easy-parse/issues/3",
			"source": "関連文書",
			"id": "github.com (issues/3)"
		},
		{
			"text": "https://cwe.mitre.org/data/definitions/611.html",
			"id": "CWE-611",
			"title": "XML 外部エンティティ参照の不適切な制限(CWE-611)"
		}
	],
	"cpe": [
		{
			"text": "cpe:/a:easy-parse_project:easy-parse",
			"version": "2.2",
			"vendor": "easy-parse project",
			"product": "easy-parse"
		}
	],
	"cvss": [
		{
			"version": "3.0",
			"score": "7.5",
			"type": "Base",
			"severity": "High",
			"vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H"
		}
	],
	"date": "2023-12-22T17:19+09:00",
	"issued": "2023-12-22T17:19+09:00",
	"modified": "2023-12-22T17:19+09:00"
}
